lib/ansible/config/base.yml

```diff
@@ -2036,6 +2036,19 @@ TASK_TIMEOUT:
   - {key: task_timeout, section: defaults}
   type: integer
   version_added: '2.10'
+_TEMPLAR_SANDBOX_MODE:
```

Comment:
why the `_` prefix, this does not seem like an 'internal ' setting but something here for backwards compatibility?